src/common/darktable.c

```diff
@@ -259,15 +259,17 @@ static void strip_semicolons_from_keymap(const char *path)
     while(c != '\n') c = fgetc(fin);
   }
 
-  // Then ignore the first two characters of each line, copying the rest out
+  
   while(c != EOF)
   {
-    fseek(fin, 2, SEEK_CUR);
-    do
+    //Ignore the first two characters of each line, copying the rest out
+    c = fgetc(fin);
+    c = fgetc(fin);
+    while((c = fgetc(fin)) != EOF)
     {
-      c = fgetc(fin);
-      if(c != EOF) fputc(c, fout);
-    } while(c != '\n' && c != EOF);
+      fputc(c, fout);
+      if(c == '\n') {break;} 
+    }
```

Comment:
what's the problem with this code, win miss fseek  ? is that something that is generic and is needed for other arch ?